### PR TITLE
Initial excision of DMTCP from Trick. #28

### PR DESCRIPTION
**Pull request by jmpenn**
This completely removes DMTCP from Trick.

### CODE REVIEW

File: autoconf/configure.ac [hunk 1]
```diff
@@ -571,13 +527,12 @@ AC_SUBST([UDUNITS_EXCLUDE])
...
 AX_GTEST_HOME([])
 AX_ER7_UTILS([])
 
-AC_OUTPUT([share/trick/makefiles/config_user.mk])
+AC_CONFIG_FILES([share/trick/makefiles/config_user.mk])
+AC_OUTPUT
+
+
 
-printf "\033@<:@32mconfigure script successfully completed\033@<:@0m\n"
```
Comment (spfennell) [inline autoconf/configure.ac:583]: Is there a reason for removing this print statement?

File: trick_source/java/src/main/java/trick/simcontrol/SimControlApplication.java [hunk 2]
```diff
@@ -995,13 +971,8 @@ private JPanel createCommandsPanel() {
...
         GridLayout gridLayout = null;
-        if (!isDmtcpOptionOn) {
-        	// 2 columns and 5 rows, each component has the same width and height.
-            gridLayout = new GridLayout(5,2,2,4);
-        } else {
-        	// 2 columns and 6 rows
-            gridLayout = new GridLayout(6,2,2,4);
-        }
+        // 2 columns and 5 rows, each component has the same width and height.
+        gridLayout = new GridLayout(5,2,2,4);
```
Comment (spfennell) [inline trick_source/java/src/main/java/trick/simcontrol/SimControlApplication.java:975]: Suggestion: 
replace
```java
GridLayout gridLayout = null;
gridLayout = new GridLayout(5,2,2,4);
```
with
```java
GridLayout gridLayout = new GridLayout(5,2,2,4);
```

**Review by spfennell — COMMENTED** [review]
It might be a good idea to test Sim Control Panel manually and make sure there isn't any layout problems before merging.

File: autoconf/configure.ac (hunk 1 above)
Comment (jmpenn) [reply autoconf/configure.ac:583]: Nope! There is not good reason for removing the print statement. Forgot to add it back in. I'll do that.

**Pull request merged**